corehq/apps/translations/generators.py

```diff
@@ -273,17 +273,10 @@ def _build_translations(self):
 
     @property
     def metadata(self):
-        if settings.TRANSIFEX_DETAILS:
-            team = settings.TRANSIFEX_DETAILS['teams'][self.domain].get(self.source_lang)
-        else:
-            team = ""
         now = str(datetime.datetime.now())
         return {
             'App-Id': self.app_id,
             'PO-Creation-Date': now,
-            'Language-Team': "{lang} ({team})".format(
-                lang=self.key_lang, team=team
-            ),
```

Comment:
is this not needed any more?

Reply:
Yup. Transifex has teams managed on its interface.